server/package.json

```diff
@@ -28,5 +37,19 @@
     "graphql-import": "^0.7.1",
     "graphql-iso-date": "^3.6.1",
     "graphql-tools": "^4.0.5"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.(js|jsx|ts|tsx)$": "<rootDir>/node_modules/ts-jest"
+    },
+    "globals": {
+      "ts-jest": {
+        "diagnostics": false
+      }
+    },
+    "reporters": [
+      "default",
+      "jest-junit"
```

Comment:
This reported is used to support jenkins style format. We use circleci so it may not be needed.